site/content/overview/gateway-architecture.md

```diff
@@ -19,7 +19,7 @@ NGINX Gateway Fabric is an open source project that provides an implementation o
 
 For a list of supported Gateway API resources and features, see the [Gateway API Compatibility]({{< relref "/overview/gateway-api-compatibility.md" >}}) documentation.
 
-We have more information regarding our [design principles](https://github.com/nginxinc/nginx-gateway-fabric/blob/main/docs/developer/design-principles.md) in the project's GitHub repository.
+We have more information regarding our [design principles](https://github.com/nginxinc/nginx-gateway-fabric/blob/v1.1.0/docs/developer/design-principles.md) in the project's GitHub repository.
```

Comment:
This feels like it can stay on the main branch, since it's not really a per-release thing.

Reply:
If we reorginze the repo folders, this link if pointing to main will suddenly become 404 on the website.

Reply:
Fair